AntistasiOfficial.Altis/UI/defines/menu_admin.hpp

```diff
@@ -16,8 +16,10 @@ else {server setVariable ['enableWpnProf',true,true]; (_this select 0) ctrlSetTe
 #define ADMIN_TGLTM "server setVariable ['testMode', [true, false] select (server getVariable ['testMode', false]), true]; \
 	(_this select 0) ctrlSetTextColor ([[1,0,0,1], [0.18,0.545,0.341,1]] select (server getVariable ['testMode', false]))" \
 
+#define ADMIN_TGLPN "if(server getVariable [""hardMode"",false]) then {server setvariable [""hardmode"",flase,true]; (_this select 0) ctrlSetTextColor [1,0,0,1];} else {server setvariable [""hardmode"",true,true];(_this select 0) ctrlSetTextColor [0.18,0.545,0.341,1];};"
+
```

Comment:
This is working partially, and missing the colour switch as line 3 of this file

Reply:
Initial button coloring looks like happening somewhere here https://github.com/A3Antistasi/antistasiofficial/blob/99447d9a6bb629c24291721c2e9c59b148f2a05b/AntistasiOfficial.Altis/UI/functions/fn_UI_createMenu.sqf#L71-L81